_stbt/pylint_plugin.py

```diff
@@ -80,6 +86,30 @@ def visit_callfunc(self, node):
                 else:
                     self.add_message('E7003', node=node, args=arg.as_string())
 
+        if _in_frameobject(node):
+            for funcdef in node.func.infer():
+                if (isinstance(funcdef, FunctionDef) and
+                        funcdef != YES and
+                        "frame" in funcdef.argnames()):
+                    index = funcdef.argnames().index("frame")
+                    if node.keywords:
+                        kwargs = [k.arg for k in node.keywords]
+                    else:
+                        kwargs = []
+                    if len(node.args) <= index and "frame" not in kwargs:
```

Comment:
We would then have to make `DeviceUnderTest` public.  Hmm, could be reasonable, not sure what it would look like.  Could provide a bunch of wrapping methods like `press()` and `frames()`, or just provide access to a `Remote` and a `Display` object.